src/app/sidebar/sidebar.component.html

```diff
@@ -7,7 +7,7 @@
       </a>
       <a [ngClass]="{'active': currentMenu === 'comment-periods'}" class="project-menu" [routerLink]="['p', currentProjectId, 'comment-periods']"><i class="material-icons">comment</i>Comment Periods</a>
       <a [ngClass]="{'active': currentMenu === 'project-surveys'}" class="project-menu" [routerLink]="['p', currentProjectId, 'project-surveys']"><i class="material-icons">format_list_bulleted</i>Surveys</a>
-      <a [ngClass]="{'active': currentMenu === 'project-documents'}" class="project-menu" (click)="goToDocuments(currentProjectId)" [routerLink]="['p', currentProjectId, 'project-documents']"><i class="material-icons">event_note</i>Documents</a>
+      <a [ngClass]="{'active': currentMenu === 'project-files'}" class="project-menu" (click)="goToDocuments(currentProjectId)" [routerLink]="['p', currentProjectId, 'project-files']"><i class="material-icons">insert_drive_file</i>Files</a>
```

Comment:
what is the purpose of the insert_drive_file inside the icon?

Reply:
In Angular's version of the Material UI framework there is a mat-icon component. That text is how you select the desired icon: https://material.angular.io/components/icon/overview